aa-sdk/core/src/client/updateHeaders.ts

```diff
@@ -0,0 +1,91 @@
+import { TraceHeader } from "../utils/traceHeader.js";
+
+/**
+ * The symbol that is used to add a breadcrumb to the headers.
+ *
+ * @see headersUpdate
+ */
+export const ADD_BREADCRUMB = Symbol("addBreadcrumb");
+
+/**
+ * The header that is used to track the trace id.
+ *
+ * @see headersUpdate
+ */
+export const TRACKER_HEADER = "X-Alchemy-Trace-Id";
+
+/**
+ * The header that is used to track the breadcrumb.
+ *
+ * @see headersUpdate
+ */
+export const TRACKER_BREADCRUMB = "X-Alchemy-Trace-Breadcrumb";
```

Comment:
these seem alchemy specific

Reply:
That would make more sense. Yeah I'll move the alchemy sepecifc ones, and then leave the optional add breadcrumbs operator in the core.